index.js

```diff
@@ -679,6 +786,29 @@ Peer.prototype._onInterval = function () {
 Peer.prototype._onSignalingStateChange = function () {
   var self = this
   if (self.destroyed) return
+
+  if (self._pc.signalingState === 'stable') {
+    self._isNegotiating = false
+    self._initialNegotiation = false
```

Comment:
`_initialNegotiation` is set, but never used. Any plans for this property?

Reply:
It was needed for automatic renegotiation, which was removed.